### DISCUSSION TITLE
Save Remote Server Password

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-jupyter#13670] [Questions and Answers] [answered]
I'm running Jupyter inside of WSL2 and set the Jupyter-Extension up to connect to this server, unfortunately the password is not stored and every time I open a new window I'm greeted with this dialog: 

![image](https://github.com/microsoft/vscode-jupyter/assets/122267262/fb1dd545-0bf3-4f09-a185-3eeb20c56bb5)

Is it possible to store the password for remote connections?

**Comment by DonJayamanne** [comment, marked as answer]
Thank you for the suggestion, this is something we have discussed internally as well
Created an issue to track this feature request
Please check this issue for updates https://github.com/microsoft/vscode-jupyter/issues/13671
